holdings_maintenance/pds4/pds4archives.py

```
@@ -0,0 +1,574 @@
+
+    dirpath = pdsdir.abspath
+
+    logger = logger or pdslogger.PdsLogger.get_logger(LOGNAME)
+    logger.replace_root(pdsdir.root_)
+
+    merged_limits = LOAD_DIRECTORY_INFO_LIMITS.copy()
+    merged_limits.update(limits)
+    logger.open('Generating file info', dirpath, limits=merged_limits)
+
+    try:
+        (tarpath, lskip) = pdsdir.archive_path_and_lskip()
```

Ruff (0.14.5)

Unpacked variable `tarpath` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)